src/components/dateUtils.js

```diff
@@ -0,0 +1,32 @@
+/* global ProcessMaker*/
+import moment from 'moment-timezone';
```

Comment:
It looks like you'll have to install `moment-timezone`, I don't see it under dependencies.